divisible.rb

```diff
@@ -0,0 +1,15 @@
+def array_of_numbers
+    (1..100).each do |n|
+      if n.even?
+        puts "#{n} is divisible by 2"
```

Comment:
You wrapped everything in a method, which is good -- the only part missing now is the "returns in an array" bit. If you ran this at the end:

```ruby
result = array_of_numbers
puts result
puts result.class
```

you'd see that your method only returns `(1..100)`, which is a `Range` instead of an `Array`.

However, because this is from a few weeks ago, I see that you already used `Array#push` in hangman.rb so I know you can do it, I might as well just tell you what you were expected to do, which is something like:

```suggestion
    array = []
    (1..100).each do |n|
      if n.even?
        array.push(n)
```

and applied to the `n % 3` and `n % 5` cases as well. Of course, you can take advantage of the `||` operator and do something like

```suggestion
    array = []
    (1..100).each do |n|
      if n.even? || n % 3 == 0 || n % 5== 0
        array.push(n)
```